### PR TITLE
Fix test_objects generation

### PR DESCRIPTION
**Pull request by agrare**
Fix a few issues with generating test_objects for specs

### CODE REVIEW

File: spec/vcr_cassettes/manageiq/providers/openshift/container_manager/test_objects_record.sh [hunk 1]
```diff
@@ -45,7 +45,7 @@ for IND in 0 1 2; do
   oc new-project my-project-$IND
   oc project my-project-$IND
   # Some objects here require the admin priviledges we got above.
-  oc process -f "$VCR_DIR"/test_objects_template.yml -v INDEX=$IND | oc create -f -
+  oc process -f "$VCR_DIR"/test_objects_template.yml INDEX=$IND | oc create -f -
```
Comment (agrare) [inline spec/vcr_cassettes/manageiq/providers/openshift/container_manager/test_objects_record.sh]: This was causing:

```
Error: invalid argument "INDEX=0" for "-v, --v" flag: strconv.Atoi: parsing "INDEX=0": invalid syntax
```

File: spec/vcr_cassettes/manageiq/providers/openshift/container_manager/test_objects_template.yml [hunk 2]
```diff
@@ -99,7 +99,7 @@ objects:
       sourceStrategy:
         from:
           kind: "DockerImage"
-          name: "ruby-20-centos7:latest"
```
Comment (agrare) [inline spec/vcr_cassettes/manageiq/providers/openshift/container_manager/test_objects_template.yml]:
```
pulling image error : repository docker.io/ruby-20-centos7 not found: does not exist or no pull access
error: build error: unable to get ruby-20-centos7:latest
```

File: spec/vcr_cassettes/manageiq/providers/openshift/container_manager/test_objects_template.yml [hunk 3]
```diff
@@ -99,7 +99,7 @@ objects:
...
+          name: "openshift/ruby-22-centos7:latest"
```
Comment (agrare) [inline spec/vcr_cassettes/manageiq/providers/openshift/container_manager/test_objects_template.yml]: Had to update to ruby 2.2 for the assemble to succeed

**Comment by cben** [comment]
LGTM :+1:

**Comment by simaishi** [comment]
Hammer backport details:

```
$ git log -1
commit 8d509c0a6be052933c5d92c72cc3184ee84ae3c6
Author: Beni Cherniavsky-Paskin <cben@redhat.com>
Date:   Fri Sep 20 10:12:46 2019 +0300

    Merge pull request #145 from agrare/fix_test_objects_generation
    
    Fix test_objects generation
    
    (cherry picked from commit 4da9f1d8990e81b9f08715372ee496a9ec9797ac)
    
    https://bugzilla.redhat.com/show_bug.cgi?id=1754071
```

**Comment by simaishi** [comment]
Ivanchuk backport details:

```
$ git log -1
commit fbffd073d6ea82c885956fed4b5c9a8db8a7f8b1
Author: Beni Cherniavsky-Paskin <cben@redhat.com>
Date:   Fri Sep 20 10:12:46 2019 +0300

    Merge pull request #145 from agrare/fix_test_objects_generation
    
    Fix test_objects generation
    
    (cherry picked from commit 4da9f1d8990e81b9f08715372ee496a9ec9797ac)
    
    https://bugzilla.redhat.com/show_bug.cgi?id=1767834
```

**Pull request merged**